andela/bp-esa-frontend
# #164237466 Implement feature to retry automations

#### What does this PR do?
Gives the ability for a failed automation to be retried

#### Description of Task to be completed?
- Set up the button component
- Set up the redux actions, sagas, and middlewares

#### How should this be manually tested?
1. Start up both the front and back-end of the app
2. On front-end:
* On the card view, click on the retry button associated with an automation card to retry the automation
* On the table view, click on the icon that pops up the modal with details of an automation. Click on the retry button to retry that automation. 
you should get a success message after 3 seconds of retrying.



#### Any background context you want to provide?
* The back-end API for this task is not yet ready, so this consumes only a mock, static data as a response. this means the id of a real automation never matches with that of the mock response.

#### What are the relevant pivotal tracker stories?
[164237466](https://www.pivotaltracker.com/story/show/164237466)

#### Postman Documentation


#### Screenshots (If applicable)
![2019-04-18 08 37 59](https://user-images.githubusercontent.com/29521319/56344977-97c93980-61b6-11e9-8719-f6781812e55a.gif)
![2019-04-18 08 38 38](https://user-images.githubusercontent.com/29521319/56344997-a283ce80-61b6-11e9-88f0-3c1a902eb375.gif)
